docs/sources/mimir/query/query-metric-labels.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Learn how to query metric labels from Grafana Mimir.
+title: Query metric labels
+menuTitle: Query metric labels
+weight: 2
+keywords:
+  - labels
+---
+
+# Query metric labels
+
+Grafana Mimir supports multiple [HTTP API]({{< relref "../references/http-api/index.md" >}}) endpoints to query metric label names and values.
+In this document we explain the main differences and trade-off between these API endpoints.
```

Comment:
```suggestion
There are differences and trade-off between these API endpoints.
```

Reply:
Rationale: Eliminate talking about the meta layer of documentation; convey the concepts directly.